nxdrive/gui/api.py

```diff
@@ -520,7 +522,7 @@ def web_update_token(self, uid: str, /) -> None:
             )
             self.setMessage.emit("CONNECTION_UNKNOWN", "error")
 
-    def _has_valid_ssl_certificate(self, server_url: str, /) -> bool:
+    def _has_valid_ssl_certificate(self, server_url: str, /) -> str:
```

Comment:
Could you rename that method to have a more explicit name? It now returns an error label. Maybe something like `_get_ssl_error()` would more suited?